src/bilder/images/odl_video_service/files/.env.tmpl

```diff
@@ -0,0 +1,75 @@
+VERSION={{ file "/etc/default/ovs-version" }}
+
+{{ with secret "postgres-odl-video-service/creds/app" }}
+DATABASE_URL="postgres://{{ .Data.username }}:{{ .Data.password }}@{{ keyOrDefault "ovs/database_endpoint" "missing_value_database_endpoint" }}/odlvideo"
+{{- end }}
+
+{{ with secret "secret-operations" }}
+CLOUDFRONT_KEY_ID="{{ index .Data "global.cloudfront-private-key.id" }}"
+CLOUDFRONT_PRIVATE_KEY="{{ index .Data "global.cloudfront-private-key.value" }}"
+MAILGUN_KEY="{{ index .Data "global.mailgun-api-key.value" }}"
```

Comment:
Is this a syntax that we can use to abstract the difference between kv-v1 and kv-v2?

Reply:
we could? but that isn't formally what it is for. This was because of the hyphens in the key names. The normal dot notation would not work.